test/test_helper.rb

```diff
@@ -67,6 +67,7 @@ def without_metadata_denormalisation(*klasses, &block)
 
   def stub_register_published_content
     WebMock.stub_request(:put, %r{\A#{PANOPTICON_ENDPOINT}/artefacts/})
+    WebMock.stub_request(:post, %r{search.dev.gov.uk/documents})
```

Comment:
This is weird, I think Plek should make us use `search.test.gov.uk` in tests. Are we setting it to `dev.gov.uk` somewhere?